common/src/main/java/com/wynntils/features/inventory/search/WynnItemFilterManager.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.inventory.search;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+import net.minecraft.client.resources.language.I18n;
+
+public class WynnItemFilterManager implements WynnItemFilterFactory {
```

Comment:
This, and the entire search package should move to `services/itemfilter`. Turn this class into a `Service` and register it (check other services to see how). Then you can access the method to create filters like this: `Services.ItemFilter.createFilter()`.